lib/xcode.js

```diff
@@ -156,12 +156,19 @@ async function getVersion (parse = false, retries = DEFAULT_NUMBER_OF_RETRIES, t
 async function getCommandLineToolsVersion () {
   let stdout;
   try {
-    stdout = (await exec('pkgutil', [`--pkg-info=com.apple.pkg.CLTools_Executables`])).stdout;
+    // first try to get the name of the correct package
```

Comment:
this looks a bit too hard to read with so many nested blocks. Can we change the stuff to be executed in a loop

```

let stdoutGetters = [
async () => (await exec('pkgutil', [`--pkg-info=${pkg.trim()}`])).stdout,
....
for (let getter of stdoutGetters) {
    try {
       return await getter();
    } catch (e) {
       // try next
    }
}
]

```

Reply:
Sure. That looks nice. Updated.